Gems/AWSGameLift/cdkv1/README.md

```diff
@@ -0,0 +1,102 @@
+
+# Welcome to O3DE GameLift Sample Project!
```

Comment:
maybe we should add a note to the top if this file about this being the old project / see `cdk` folder for newer CDKv2 version

Reply:
Sure can add that.